main/src/addins/VersionControl/MonoDevelop.VersionControl/MonoDevelop.VersionControl/DefaultBlameViewHandler.cs

```diff
@@ -36,7 +36,7 @@ public class DefaultBlameViewHandler : IVersionControlViewHandler
 	{
 		public bool CanHandle (VersionControlItem item, DocumentController controller)
 		{
-			return (controller is FileDocumentController) && item.Repository.GetFileIsText (item.Path);
+			return (controller is FileDocumentController || controller == null) && item.Repository.GetFileIsText (item.Path);
```

Comment:
```suggestion
			return (controller is FileDocumentController || controller is null) && item.Repository.GetFileIsText (item.Path);
```

Minor improvement in case someone ends up adding an `operator ==` to DocumentController.